doc-content/enterprise-only/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-console-custom-image-build.adoc

```diff
@@ -7,6 +7,19 @@ The operator uses the following custom resources to deploy domain-specific micro
 
 * `KogitoRuntime` starts the runtime image and configures it as per your requirements.
 
+[NOTE]
+====
+The {PRODUCT} builder image does not supports native builds.  However, you can perform a custom build and use `Containerfile` to build the container image as shown in the following example:
+
+`FROM registry.redhat.io/rhpam-7-tech-preview/rhpam-kogito-runtime-native-rhel8:{ENTERPRISE_VERSION_LONG}`
+
+`ENV RUNTIME_TYPE quarkus`
+
```

Comment:
Seems that the COPY command is missing.

something like:

```
# To build the native binary with Mandrel please visit <doc to Quarkus native builds>
COPY --chown=1001:root target/*-runner $KOGITO_HOME/bin
```

Reply:
I am ok with that